front/src/modules/ui/feedback/dialog/types/DialogOptions.ts

```diff
@@ -0,0 +1,5 @@
+import { DialogProps } from '../components/Dialog';
+
+export type DialogOptions = DialogProps & {
+  id: string;
```

Comment:
what is is?

Reply:
It is used to type the elements in the queue